src/content/tutorials/connectivity/coredns/index.md

```diff
@@ -4,59 +4,58 @@ title: Advanced CoreDNS configuration
 description: Here we describe how you can customize the configuration of the managed CoreDNS service in your clusters.
 weight: 25
 menu:
-  main:
-    parent: advanced-connectivity
+  principal:
+    parent: tutorials-connectivity
+    identifier: tutorials-connectivity-coredns
+aliases:
+  - /advanced/connectivity/coredns
+  - /guides/advanced-coredns-configuration/
+  - /advanced/coredns/
 user_questions:
   - How can I override the default CoreDNS configuration?
   - How can I customize the CoreDNS configuration?
   - How can I adjust resource limits for CoreDNS?
   - Where is the user values ConfigMap for CoreDNS?
-last_review_date: 2023-11-07
-aliases:
-  - /advanced/connectivity/coredns
-  - /guides/advanced-coredns-configuration/
-  - /advanced/coredns/
+last_review_date: 2025-07-07
 owner:
   - https://github.com/orgs/giantswarm/teams/team-phoenix
 ---
 
-Your Giant Swarm installation comes with a default configuration for the [CoreDNS addon](https://github.com/coredns/coredns)
+Your Giant Swarm clusters come with a default configuration for the [CoreDNS addon](https://github.com/coredns/coredns). CoreDNS is installed as default application in your clusters using a HelmRelease named `<CLUSTER_ID>-coredns`. This guide explains how you can customize the CoreDNS configuration in your clusters.
 
-You can override these defaults in a ConfigMap named `coredns-user-values` in the management cluster.
+## Where to store the user configuration
 
-__Note__: In Cluster API, CoreDNS is part of default apps `<provider>` chart (example [Cluster API for Openstack](https://github.com/giantswarm/default-apps-openstack/). In order to extend the app, you need to patch the App Custom Resource to use `coredns-user-values` configmap. [Check out this example](https://github.com/giantswarm/gitops-template/pull/74/files).
+Given the cluster you are trying to configure has id: `123ab` then you will find a `123ab-coredns` HelmRelease in the organization namespace of the management cluster. Inside this HelmRelease, there is a field called `ValuesFrom` which lets you define a set of ConfigMaps that will be used to override the default values of the CoreDNS Helm chart.
```

Comment:
@giantswarm/team-cabbage I need a bi of help to write this part PTAL

Reply:
can you facilite an example plz?

Reply:
Customers can add the following values to their `cluster-userconfig` configmap:
```
    global:
        apps:
            coreDns:
                extraConfigs:
                    - name: <cluster_id>-coredns-cluster-values
                      kind: ConfigMap
```

but this is certainly not specific of CoreDNS. I think we should move that to a generic "how to configure any default app" tutorial.

Reply:
done